lib/src/builder.dart

```diff
@@ -333,9 +333,38 @@ class DartTestYamlBuilder extends Builder {
         AssetId(buildStep.inputId.package, 'dart_test.browser_aggregate.yaml');
     await buildStep.writeAsString(outputId, contents.toString());
 
-    final backwardsCompatOutputId = AssetId(
-        buildStep.inputId.package, 'test/dart_test.browser_aggregate.yaml');
-    await buildStep.writeAsString(backwardsCompatOutputId, contents.toString());
+    var shouldWriteBackwardsCompatOutput = false;
+    try {
+      final dartTestYamlId =
+          AssetId(buildStep.inputId.package, 'dart_test.yaml');
+      final dartTestYaml = await buildStep.readAsString(dartTestYamlId);
+      if (dartTestYaml.contains('test/dart_test.browser_aggregate.yaml')) {
+        log.fine(
+            'Found `test/dart_test.browser_aggregate.yaml` in `dart_test.yaml`, will generate it for backwards-compatibility.');
```

Comment:
**Suggestion**: What would you think about making this an info or warning log that includes a suggestion to update the dart_test.yaml file to reference the new file?